canonical-web-and-design/ubuntu.com
# add version specific esm and livepatch instructions to /advantage

## Done
Updated the instructions in the ESM and Livepatch cells on /advantage, according to the changes made [here](https://docs.google.com/document/d/1K65n3NpZCh7EWDXKLhG31seGJ2jKbW6lqw3BwV2mlnU/edit#heading=h.dqabkhv1pde3)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/advantage
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Login with SSO
- If you don't have already have any enterprise subscriptions, go to /advantage/subscribe and buy one with test card details
  - Card number: 4242 4242 4242 4242, expiry: 04/24, cvc: 242, zip: 42424
- back on /advantage, in your subscription, expand the ESM and Livepatch cells, and see that they match the copy in [the doc](https://docs.google.com/document/d/1K65n3NpZCh7EWDXKLhG31seGJ2jKbW6lqw3BwV2mlnU/edit#heading=h.dqabkhv1pde3)


## Issue / Card

Fixes #

## Screenshots

[If relevant, please include a screenshot.]
